frontend/src/v5/ui/controls/avatar/avatar.styles.ts

```diff
@@ -18,41 +18,48 @@
 import styled, { css } from 'styled-components';
 import { IconButton } from '@mui/material';
 
-export const StyledIconButton = styled(IconButton)<{ $isButton?: boolean, disabled?: boolean, $largeIcon?: boolean }>`
-	&& {
-		padding: 0;
-		margin: 0;
+const getSizeInPixels = (size?: string) => {
+	if (size === 'large') return '115px';
+	if (size === 'medium') return '48px';
+	// 'small' or default
+	return '38px';
+};
+
+export const StyledIconButton = styled(IconButton)<{ $isButton?: boolean, disabled?: boolean, size?: string }>`
+	padding: 0;
+	margin: 8px 7px;
 
-		${({ $isButton, theme }) => ($isButton ? `
-			cursor: pointer;
-			
-			&:hover {
-				.MuiAvatar-root {
-					background-color: ${theme.palette.tertiary.mid};
-				}
+	${({ $isButton, theme }) => ($isButton ? `
+		cursor: pointer;
+		
+		&:hover {
+			.MuiAvatar-root {
+				background-color: ${theme.palette.tertiary.mid};
 			}
+		}
 
-			&:active {
-				.MuiAvatar-root {
-					background-color: ${theme.palette.tertiary.main};
-				}
+		&:active {
+			.MuiAvatar-root {
+				background-color: ${theme.palette.tertiary.main};
 			}
-		` : `
-			cursor: default;
-		`)}
+		}
+	` : 'cursor: default;')}
+
+	&:hover {
+		background-color: transparent;
 	}
 
-	.MuiAvatar-circle {
+	.MuiAvatar-circular {
 		margin: 0;
-		${({ $largeIcon }) => `
-			height: ${$largeIcon ? '48px' : '38px'};
-			width: ${$largeIcon ? '48px' : '38px'};
+		${({ size }) => `
+			height: ${getSizeInPixels(size)};
+			width: ${getSizeInPixels(size)};
```

Comment:
I think we should also increase/decrease the font size to match the style
<img src="https://user-images.githubusercontent.com/42032433/176156293-ca705cb6-4f1d-436a-b5c3-65f5576ec886.png" width="300"/>